### PR TITLE
Add DYCOMS_RF02 case

### PR DESCRIPTION
**Pull request by trontrytel**
This adds a drizzling Sc test case. Currently the LES data is generated with SB microphysics scheme. Once I'm done with implementing the CliMA 1-moment scheme we should switch the dataset

### CODE REVIEW

File: integration_tests/DYCOMS_RF02.jl [hunk 1]
```diff
@@ -0,0 +1,41 @@
+if !("." in LOAD_PATH) # for easier local testing
+    push!(LOAD_PATH, ".")
+end
```
Comment (charleskawczynski) [inline integration_tests/DYCOMS_RF02.jl:3]:
```suggestion
import Pkg
Pkg.develop(path = ".")
```

**Review by charleskawczynski — APPROVED** [review]
LGTM.

This would be a great PR to document, somehow, since it cleanly adds a new experiment, and all of the separate pieces that are needed.

**Comment by trontrytel** [comment]
> LGTM.
> 
> This would be a great PR to document, somehow, since it cleanly adds a new experiment, and all of the separate pieces that are needed.

Should I write down the commentary as a comment to this PR?

**Comment by charleskawczynski** [comment]
> Should I write down the commentary as a comment to this PR?

I think it's fine, we can easily grab the commit if/when we add such docs

**Review by yairchn — APPROVED** [review]
LGTM

**Pull request merged**